src/app/navbar/navbar.component.html

```diff
@@ -14,10 +14,16 @@
     <a class="navbar-item">
       <span [ngClass]="ping_info.is_alive?'tag is-success':'tag is-danger'">{{ping_info.service_status}}</span>
     </a>
+    <a class="navbar-item">
+      <button *ngIf="ping_info.is_alive" class="button is-danger is-small" (click)="openModal()">
+        <i class="fa fa-power-off" aria-hidden="true"></i>
+      </button>
+    </a>
   </div>
   <div class="navbar-burger" (click)="toggleClick()">
       <span></span>
       <span></span>
       <span></span>
   </div>
+  <app-modal (shutdown)='shutdown($event)' [childData]='childData'></app-modal>
```

Comment:
shutdown-foglamp and have corresponding emitter so that it does not interfere with others;